src/main/webapp/app/admin/system-notification-management/system-notification-management.route.ts

```diff
@@ -23,50 +23,63 @@ export class SystemNotificationManagementResolve implements Resolve<any> {
     }
 }
 
-export const systemNotificationManagementRoute: Route = {
-    path: 'system-notification-management',
-    component: SystemNotificationManagementComponent,
-    resolve: {
-        pagingParams: JhiResolvePagingParams,
-    },
-    data: {
-        pageTitle: 'artemisApp.systemNotification.systemNotifications',
-        defaultSort: 'id,asc',
+export const systemNotificationManagementRoute: Route[] = [
+    {
+        path: 'system-notification-management',
+        component: SystemNotificationManagementComponent,
+        resolve: {
+            pagingParams: JhiResolvePagingParams,
+        },
+        data: {
+            pageTitle: 'artemisApp.systemNotification.systemNotifications',
+            defaultSort: 'id,asc',
+        },
     },
-    children: [
-        {
-            path: 'new',
-            component: SystemNotificationManagementUpdateComponent,
-            data: {
-                pageTitle: 'global.generic.create',
-            },
+    {
+        // Create a new path without a component defined to prevent resolver the SystemNotificationManagementComponent from being always rendered
```

Comment:
Assume this is the same comments as above removed?
```suggestion
        // Create a new path without a component defined to prevent resolver caching and the SystemNotificationManagementComponent from being always rendered
```